### ISSUE TITLE
Invalid Call : Nonexistent function 'snapshotsSaveGame' in base 'JNISingleton'

### ISSUE THREAD

**Issue by suhid98** [Iakobs/godot-google-play-game-services-android-plugin#11]
I am using Godot 3.5 and downloaded both the plugin for android and the helper Godot plugin directly from asset library ( also tried manually). 

The sign in successfully works. So does the "snapshots_load_game" function. But when I want to save game this error occurs!

![image](https://github.com/Iakobs/godot-google-play-game-services-android-plugin/assets/25875304/31b38b1d-0926-4605-ab40-f5beb400a2e9)

**Comment by Iakobs** [comment]
Hi @Faisul-Islam thanks for you question!

This error can happen when the method you're trying to call doesn't exist, it's misspelled, or the parameters provided are wrong, but this case is strange, since you say that the load save game method is working.

I was thinking of suggesting checking the version of the plugin you're using, because saved games feature was introduced the latest, but if you have loading, saving should work also.

I checked spelling and arguments and they look correct, too. My last suggestion would be trying the godot plugin for this library, [here](https://github.com/Iakobs/godot-google-play-game-services-plugin/tree/main). The plugin is doing basically the same you do (check it [here](https://github.com/Iakobs/godot-google-play-game-services-plugin/blob/main/addons/google_play_game_services_manager/singleton.gd#L258)), but it also contains a demo project that might help you further.

Please, let me know if this helped you!

**Comment by suhid98** [comment]
Isn't [this repo](https://github.com/Iakobs/godot-google-play-game-services-plugin/tree/main) that you linked the same helper godot plugin that can be downloaded directly from asset library? Just the example folder stripped.

Anyway, one another thing is that, weirdly I also got errors of nonexistent signal's for these two signal ( which I have commented out ). After commenting that actually I am getting the error of the issue I opened Like I mentioned in OP.
**![image](https://github.com/Iakobs/godot-google-play-game-services-android-plugin/assets/25875304/bdc75367-367e-4b01-ade1-7c4195aba141)**

And, here is the plugin version : 
![image](https://github.com/Iakobs/godot-google-play-game-services-android-plugin/assets/25875304/90329f2a-60b1-49c6-a122-2a3810d26863)
